data/campaigns/fri05.wmf/scripting/texts.lua

```diff
@@ -0,0 +1,371 @@
+      .. paragraphdivider() ..
+      -- TRANSLATORS: Henneke Lembeck – Training
+      _([[At all times, we will need to keep a close eye on our supply of tradeable goods. I will ask Ocke Ketelsen to purchase only the goods our economy actually needs so that we won’t waste any wares, but it will be up to us to ensure we can provide enough goods. These trade routes are all very expensive, and we will need to mark out a large amount of space purely to produce wares to sell.]])),
+   position = "topleft",
+}
+
+diking = {
+   title = _("Planning the War"),
+   body=claus(_("Diking"),
+      -- TRANSLATORS: Claus Lembeck – Diking
+      _([[The lack of space on Helgoland is still a concern. The land would be sufficiently large to train a decent number of soldiers in a reasonable time, but training the armies we believe we will need would take forever! We may need to take some chances, and use superior tactics to compensate for this disadvantage.]])
+      .. paragraphdivider() ..
+      -- TRANSLATORS: Claus Lembeck – Diking
+      _([[However, if you prefer to play it safe, don’t forget that we can simply enlarge the island. As we have a steady supply of logs now, we can construct dikes and breakwaters to gain more building space from the sea. When we are ready to attack, we can even use this as a way of bypassing the chokepoint in the east, or if we are to be truly bold we could dike around the whole western coast of the island to attack the enemy from two sides at once!]])),
+   position = "topleft",
+}
+
+getting_started_4 = {
+   title = _("Planning the War"),
+   body=ketelsen(_("Good luck!"),
+      -- TRANSLATORS: Ocke Ketelsen – Getting Started 4
+      _([[I am heartened by your forwardness. Let us all hasten to our separate tasks, and may fortune smile on us!]]))
+}
+
+secured_wood = {
+   title = _("Wood Supply"),
+   body=henneke(_("Wood supply is ensured"),
+      -- TRANSLATORS: Henneke Lembeck – Wood Supply
+      _([[Our foresters and woodcutters have taken up their work. It will take some time before we have a steady supply of logs, but the immediate problem has been averted.]]))
+}
+
+secured_block = {
+   title = _("Pass Secured"),
+   body=claus(_("The pass is under guard"),
+      -- TRANSLATORS: Claus Lembeck – Pass Secured
+      _([[Our soldiers have fortified the pass. Surely no enemy will get past them before we are ready.]]))
+}
+
+secured_uplands = {
+   title = _("Uplands Colonized"),
+   body=reebaud(_("The Uplands have been settled"),
+      -- TRANSLATORS: Reebaud – Uplands Secured
+      _([[The Uplands are being converted to a true center of our economy. We must not forget to always keep a close eye on the ware exchange between Lowlands and Uplands and keep it to a minimum so as to avoid jams on the roads.]]))
```

Comment:
```suggestion
      _([[The Uplands are being converted to a true center of our economy. We must not forget to always keep a close eye on the ware exchange between the Lowlands and the Uplands and keep it to a minimum so as to avoid jams on the roads.]]))
```
?